src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +17,51 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const apr = getConfig().REACT_APP_ETH_FOX_APR
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/eth@2x.png'
+  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/fox@2x.png'
```

Comment:
please don't hardcode these here - we have a single source of truth for assets including high quality asset icons

please use the `AssetId` for ethereum and fox, use `selectAssetById` selector, and pull the icon from the asset